components/badge/badge.css

```diff
@@ -0,0 +1,82 @@
+.badge{
+    position: relative;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 1rem;
+}
+
+.badge-icon{
+    font-size: 2rem;
+    color:gray;
+    
+}
+
+.badge-number{
+    position: absolute;
+    top: -0.5rem;;
+    right: -0.5rem;
+    display: flex;
```

Comment:
you could move the repetitive code (like flex, height, width, font-size) into separate class and use it

Reply:
right , ill try to implement it